### ISSUE TITLE
Question: how to train models with multiple targets

### ISSUE THREAD

**Issue by tobyjwebb** [smartcorelib/smartcore#84]
Hi there!

Is it possible to train models with multiple outputs?

Regards,

TW

**Comment by VolodymyrOrlov** [comment]
Hi Toby, can you share a bit more details of your model? Are you looking for an algorithm that can handle [multiclass](https://en.wikipedia.org/wiki/Multiclass_classification), [multi-label](https://en.wikipedia.org/wiki/Multi-label_classification) classification or something entirely different?

**Comment by tobyjwebb** [comment]
Is there such a thing as multi label regression?

I'm a bit of a noob regarding ML, but I take it the labels are kind of
"output neurons", right? All if the examples I've found so far seen to have
only one output value.

As an example of what I'm trying to do, I want to feed the neutral network
a series of data, and I want to try to forecast the amount of phone calls
to emergency services in different regions. The different regions would be
the output neurons / targets.

Thanks,

TW

**Comment by VolodymyrOrlov** [comment]
Toby, you might want to take a look at one of many deep learning frameworks, e.g. [Tensorflow](https://www.tensorflow.org/) or [PyTorch](https://pytorch.org/) if your goal is to build a neural network that predicts a numerical target. Smartcore provides only 'traditional' machine learning algorithms, like Random Forest or Linear Regression. I don't know how your dataset looks like but what you do with Smartcore is to transform your data into a set of features and fit [LinearRegression](https://docs.rs/smartcore/0.2.0/smartcore/linear/linear_regression/index.html) to these features. Your target will be number of calls and region will be one of predictors, for example: 

region (one-hot-encoded), number of emergency calls 1 day before, number of emergency calls 2 day before, ... -> number of emergency calls tomorrow

**Comment by Mec-iS** [comment]
see #370
